<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTable</class>
 <widget class="QDialog" name="MainTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="VToInsert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddVButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="Text1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hit percent</string>
   </property>
  </widget>
  <widget class="QLabel" name="Text2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fail percent</string>
   </property>
  </widget>
  <widget class="QLabel" name="HitPercentValue">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="FailPercentValue">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="LRUMethod">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lru</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="LFUMethod">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lfu</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="FIFOMethod">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fifo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="LIFOMethod">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lifo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="LiveTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>751</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Replacement Method</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
